mmpose/models/detectors/associative_embedding.py

```diff
@@ -74,7 +75,9 @@ def with_keypoint(self):
 
     def init_weights(self, pretrained=None):
```

Comment:
Do you mean that in MME `init_weights()` will be invoked separately after `__init__()`, which means `init_weights()` will be invoked twice during model building?

Reply:
yes

Reply:
Wouldn't that be weird since the model weights will be initialized twice?

Reply:
This is only a temporary modification. In order to be compatible with mme, we have to initialize the parameters twice. It is recommended to inherit all model components from `BaseModule` and delete `init_weights()` in the future.